README.md

```diff
@@ -113,6 +113,17 @@ It takes in a computed application object and returns an augmented score. The id
 module.exports = ({ rating }) => rating;
 ```
 
+It also makes use of a selection script stored in `src/js/hc-scripts/choose-applicants.js`
+This takes a sorted list of applications, a result count limit and an inviteType
+and returns a list of the suggested applications to invite/reject.
+E.g. to just takes the top n in the sorted list:
+
+```
+module.exports = ( (sortedApplications, n, inviteType) => {
+  return sortedApplications.slice(0, Math.min(sortedApplications.length, limit));
```

Comment:
Should this be `n` instead of `limit`? Also, if you slice, it doesn't matter if the second argument is greater than the size of the list, so you can get away with `sortedApplications.slice(0, n)`.